R/UnConstrainedFPOPLog.R

```diff
@@ -0,0 +1,227 @@
+PeakSegFPOP <- structure(function
```

Comment:
you should probably rename this here and in the examples below, to UnConstrainedFPOP